### PR TITLE
feat: add rating

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

**Review by prajapati-priyanka — APPROVED** [review]
Class names are self-explanatory.  Clean Code.

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.rating-container{
+    font-size: 3rem;
+    color: var(--code-color);
+}
+
+.number-rating{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: .5rem 1.5rem;
```
Comment (yaswanthmyneni) [inline components/rating/rating.css:10]: changes: use px for less than 1rem

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes.

**Review by Husain01 — APPROVED** [review]
Looks good to go.

File: components/rating/rating.css (hunk 1 above)
Comment (vermasavita) [reply components/rating/rating.css:10]: Noted

**Pull request merged**